frontend/src/app/containers/ExperimentPage.js

```diff
@@ -367,11 +367,11 @@ export class ExperimentDetail extends React.Component {
                           </li>
                         ))}
                       </ul>
-                    {contributors_extra && <p className="disclaimer"
-                        dangerouslySetInnerHTML={createMarkup(contributors_extra)}
-                        data-l10n-id={this.l10nId('contributors_extra')}
-                        target="_blank"
-                        href={contributors_extra_url} />}
+                    {contributors_extra && <p className="disclaimer">
+                        <span data-l10n-id={this.l10nId('contributors_extra')}>{contributors_extra}</span>&nbsp;
+                        <a data-l10n-id="contributorsExtraURL" href={contributors_extra_url} target="_blank">Learn more</a>.
```

Comment:
Is `contributors_extra_url` required? If so, we should call that out explicitly in docs. If not, it should be in a conditional here.